contracts/KyberProxy.sol

```diff
@@ -35,43 +39,46 @@ contract KyberProxy is TokenConverter, Ownable {
         uint256 minReturn
     ) external payable returns (uint256 destAmount) {
         
-        destAmount = _convert(ERC20(srcToken), ERC20(destToken), srcQty);
+        destAmount = _convert(srcToken, destToken, srcQty);
```

Comment:
If msg.sender is converting from ETH we shall check that msg.value equals srcQty